src/legacy/core_plugins/kibana/public/discover/np_ready/angular/discover.js

```diff
@@ -891,6 +880,9 @@ function discoverController(
         tabifiedData,
         getDimensions($scope.vis.data.aggs.aggs, $scope.timeRange)
       );
+      if ($scope.vis.data.aggs.aggs[1]) {
+        $scope.bucketInterval = $scope.vis.data.aggs.aggs[1].buckets.getInterval();
+      }
```

Comment:
dear @ppisljar, this API changed. This works, but I wonder if there's a better way to do it. Before it was done this way:

```
      const buckets = $scope.vis.data.aggs.byTypeName('buckets');		

          if (buckets && buckets.length === 1) {		
           $scope.bucketInterval = buckets[0].buckets.getInterval();		
         }
```